docs/sources/mimir/operators-guide/configure/about-configurations.md

```diff
@@ -59,3 +59,10 @@ There is no harm in passing a configuration that is specific to one component (s
 If you need to, you can use advanced CLI flags to override specific values on a particular Grafana Mimir component or replica. This can be helpful if you want to change a parameter that is specific to a certain component, without having to do a full restart of all other components.
 
 The most common use case for CLI flags is to use the `-target` flag to run Grafana Mimir as microservices. By setting the `-target` CLI flag, all Grafana Mimir components share the same configuration file, but you can make them behave as a given component by specifying a `-target` command-line value, such as `-target=ingester` or `-target=querier`.
+
+## Validate a configuration
+
+To validate your configuration, run the command `mimir -modules true -config.file <path-to-config-file>`.
+This is useful if you want to validate the configuration in a CI environment before deploying it.
```

Comment:
```suggestion
This is useful if you want to validate the configuration before deploying it and can be used both locally and in a CI environment.
```

@osg-grafana I'd like to rephrase this, do this look good to you?

Reply:
```suggestion
This is useful if you want to validate the configuration before deploying it. You can use the command locally or in a CI environment.
```

Reply:
The subject is missing in the second part of that passive construction. I assume you mean that you can use the command (and not the configuration file) in that second part. Please vet for technical accuracy.

Reply:
Thank you, looks good.